src/github3/structs.py

```diff
@@ -112,7 +112,12 @@ def __next__(self):
         return next(self.__i__)
 
     def _get_json(self, response):
-        return self._json(response, 200)
+        json = self._json(response, 200)
+        if isinstance(json, dict):
+            list_key = getattr(self.cls, "list_response_dict_key", None)
```

Comment:
Does this mean that GitHub has totally abandoned their design principles for APIs and now return the list like so:

```json
{
  "check_runs": [{
    ...
  }, {
    ...
  }]
}
```

If so, I would rather not have this be an attribute on the class. I'd rather pass this into the `_iter` method and make this an attribute on the iterator class.

Reply:
Yes, that's the case. Will move this into `_iter`.